flow-sample/src/main/java/com/example/flow/MainActivity.java

```diff
@@ -64,16 +64,13 @@ protected void onCreate(Bundle savedInstanceState) {
     GsonParceler parceler = new GsonParceler(new Gson());
     @SuppressWarnings("deprecation") ActivityFlowSupport.NonConfigurationInstance nonConfig =
         (ActivityFlowSupport.NonConfigurationInstance) getLastNonConfigurationInstance();
-    flowSupport = ActivityFlowSupport.onCreate(nonConfig, getIntent(), savedInstanceState, parceler,
-        Backstack.single(new Paths.ConversationList()));
-
     final ActionBar actionBar = getActionBar();
     actionBar.setDisplayShowHomeEnabled(false);
-
     setContentView(R.layout.root_layout);
-
     container = (PathContainerView) findViewById(R.id.container);
     containerAsBackTarget = (HandlesBack) container;
+    flowSupport = ActivityFlowSupport.onCreate(nonConfig, getIntent(), savedInstanceState, parceler,
```

Comment:
Should add javadoc to ActivityFlowSupport#onCreate that this should be called after setContentView.